File: python/011.py
# 11) Desenvolva uma lógica que leia os valores de A, B e C de uma equação do
# segundo grau e mostre o valor de Delta.

print("Equação de segundo 2ª Grau")
print("ax² + bx + c = 0")
a = int(input("Digite o valor de A: "))
b = int(input("Digite o valor de B: "))
c = int(input("Digite o valor de C: "))

print("Calculando delta..")
delta = b**2 -(4 * a * c)
print(f"Delta é igual a {delta}")

File: javascript/011.js
/*
11) Desenvolva uma lógica que leia os valores de A, B e C de uma equação do
segundo grau e mostre o valor de Delta.
*/

const delta = (a, b, c) => {
  const calc = b ** 2 - 4 * a * c;
  return calc;
};

console.log(delta(1, -4, 3));
